src/dlgdevelopertoolsdlg.ui

```diff
@@ -40,7 +40,17 @@
          </property>
         </spacer>
        </item>
-       <item row="1" column="0" colspan="2">
+       <item row="0" column="1">
+        <widget class="QPushButton" name="controlDump">
+         <property name="toolTip">
+          <string>Dumps all ControlObject values to a csv-file saved in the settings path (e.g. ~/.mixxx)</string>
```

Comment:
This path in the example is only valid for Linux, might be better to link to a wiki/manual page were we list the actual path for all supported platforms.